mods/d2k/rules/defaults.yaml

```diff
@@ -117,6 +117,9 @@
 		RequiresCondition: rank-elite
 		ZOffset: 256
 
+^RevealsShroudOnFire:
```

Comment:
I accept that this works like it does but I would like to know why this trait is declared here and inherited for further use?